uni/ulsp/server.icn

```diff
@@ -87,9 +87,13 @@ class Server(
                if pos(0) then {
                   request_body := ready(sock, len)
                }
-               else
+               else {
                   request_body := move(len)
-                  msg := tab(0)
+               }
+               until *request_body = len do {
```

Comment:
This had the potential of running infinitely. I recommend adding a `select()` with a 5 seconds   timeout, maybe in a loop with 1 second increment, after that, consider that a malformed msg and move on.